bears/c_languages/ClangBear.py

```diff
@@ -27,6 +27,8 @@ class ClangBear(LocalBear):
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {}
```

Comment:
it feels misleading, as you already specify can fix, and it should have the same values at CAN_DETECT as well.

by the way you are pointing it, you are specifying explicitly that it cannot detect what it fixes, which we dont want to show